src/client.js

```diff
@@ -1782,13 +1782,17 @@ class Client {
                   if (entity) {
                       const impls = DomUtil.getAttributeAsString(entity, "implements");
                       if (impls === "xtk:persist" && schemaId !== "xtk:session" && schemaId !== "xtk:persist") {
-                        // Ensure xtk:persist is present by loading the xtk:session schema
-                        await this.getSchema("xtk:session", "xml", true);
+                          // Ensure xtk:persist is present by loading the xtk:session schema
+                          await this.getSchema("xtk:session", "xml", true);
                       }
                       await this._entityCache.put("xtk:schema", schemaId, entity);
                       await this._methodCache.put(entity);
                   }
             }
+        } else {
```

Comment:
You should not reload the methods for each getSchema method, only if methods are not there in the cache or if they have expired

Reply:
The problem is that getSchema does not indicate if the schema was from the memory cache or was load in memory from the cache of the browser (local storage or cache storage), so I add the methods to cover both cases.

Reply:
Another possibility is to move `await this._methodCache.put(entity);` above in the xtk:session/xtk:persist if :
```javascript
if (impls === "xtk:persist" && schemaId !== "xtk:session" && schemaId !== "xtk:persist") {
    // Ensure xtk:persist is present by loading the xtk:session schema
    const xtkSession = await this.getSchema("xtk:session", "xml", true);
    await this._methodCache.put(xtkSession );
}
```
because it impacts only xtk:persist, xtk:session missing methods